integrations/sumologic/sumologic-analytics/sensu-resources.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       api_version: pipeline/v1
       type: SumoLogicMetricsHandler
       name: sumologic-metrics
-  - name: sumologic-logs
+  - name: sumologic-events
```

Comment:
Need to confirm this change won't disturb anything in the integration

Reply:
Does look like it will, nothing targets the specific workflow.